components/accept-financial-contributions/StartAcceptingFinancialContributionsPage.tsx

```diff
@@ -0,0 +1,582 @@
+  const currencyOptions = Currency.map(c => ({ value: c, label: c }));
+  const [selectedCurrency, setSelectedCurrency] = React.useState(currencyOptions[0]);
+  const [searchingForFiscalHost, setSearchingForFiscalHost] = React.useState(false);
+  const router = useRouter();
+  const independentCollectiveOption = router.query.independentCollectiveOption === 'true';
+
+  const countrySelectOptions = React.useMemo(() => {
+    const countryList = isoCountryList
+      .map(countryCode => ({
+        value: countryCode,
+        label: getCountryDisplayName(intl, countryCode),
+      }))
+      .sort((a, b) => a.label.localeCompare(b.label));
+
+    return [allCountriesSelectOption, ...countryList];
+  }, [intl]);
+
+  return (
+    <Flex my={5} alignItems="center" flexDirection="column">
+      <Illustration
+        alt="A place to grow and thrive illustration"
+        src="/static/images/fiscal-hosting/what-is-a-fiscalhost-illustration.png"
+      />
+      <Box mt={4} mb={4}>
+        <H1
+          fontSize={['20px', '32px']}
+          lineHeight={['24px', '40px']}
+          fontWeight="700"
+          color="black.900"
+          textAlign="center"
+        >
+          <FormattedMessage id="contributions.startAccepting" defaultMessage="Start accepting contributions" />
+        </H1>
+        <P mt={3} fontSize="16px" lineHeight="24px" fontWeight="500" color="black.700" textAlign="center">
+          <FormattedMessage defaultMessage="Choose who will hold the money on your behalf" />
+        </P>
+      </Box>
+      <StyledCard width={['300px', '400px', '600px', '800px']} padding="32px 24px">
+        <P mb={1} fontSize="16px" lineHeight="24px" fontWeight="700" color="black.800">
+          <FormattedMessage defaultMessage="What type of community are you?" />
+        </P>
+        <StyledFilters
+          multiSelect
+          filters={CommunityTypes}
+          onChange={setSelectedCommunityType}
+          selected={selectedCommunityType}
+          flexWrap="wrap"
+        />
+
+        <Flex gap="40px">
+          <Box>
+            <P mt={4} fontSize="16px" lineHeight="24px" fontWeight="700" color="black.800">
+              <FormattedMessage defaultMessage="Where would your collective be most active?" />
+            </P>
+            <StyledSelect
+              value={selectedCountry}
+              inputId="country-input"
+              options={countrySelectOptions}
+              onChange={setSelectedCountry}
+            />
+            <P color="black.600" fontWeight="400" fontSize="11px" lineHeight="16px">
+              <FormattedMessage defaultMessage="If multiple areas, please select most prominent of them all." />
+            </P>
+          </Box>
+          <Box flexGrow={1}>
+            <P mt={4} fontSize="16px" lineHeight="24px" fontWeight="700" color="black.800">
+              <FormattedMessage defaultMessage="What currency will your collective use?" />
+            </P>
+            <StyledSelect
```

Comment:
Maybe some padding on top to separate the message from the input as to me both `<StyedSelect` seemed be squished together with the label. 

![image](https://user-images.githubusercontent.com/12435965/228474467-ad0127de-4a19-4d95-a9b3-8ab4affd1c74.png)